src/components/MapView/index.js

```diff
@@ -1,9 +1,10 @@
 import React, { Component} from 'react';
-import ReactMapGL, {Marker } from 'react-map-gl';
+import ReactMapGL, {Marker, Popup } from 'react-map-gl';
 import { SERVER_URL } from '../../utils/constants';
- 
- 
- 
+import { withRouter } from 'react-router-dom';
+import defaultImage from '../../assets/location/ahmedabad.svg';
+import { mdiBed } from '../../assets/bed.svg';
```

Comment:
wrong see how other icon are implemented 
I give you the link it ill have the code,

the whole thing is not looking good, be consistent, it's all about symmetry, 

icon des: value <- follow this for everything,
icons are too black compare to text
reduce icon size so they look well